### PR TITLE
Task/arch 676

### PR DESCRIPTION
**Pull request by shafi1379**
@tfokken @rk-fsdev @satish-sanjeev Make Changes For Add Work Space Member API

### CODE REVIEW

**Review by afoerster — APPROVED** [review]
LGTM

**Comment by afoerster** [comment]
@rk-fsdev can you take a look?

File: ui/containers/WorkspaceDetails/sagas.ts [hunk 1]
```diff
@@ -209,23 +209,26 @@ export function* simpleMemberRequested({ resource }: SimpleMemberRequestAction)
...
+      if (workspaceMemberList && workspaceMemberList.length) {
+        yield call(Api.newWorkspaceMember, token, workspaceMemberList[0].resource_id, workspaceMemberList);
+      }
       yield put(simpleMemberRequestComplete());
     } else if (resource === 'topics') {
       const { id } = yield select(activeTopicExtractor);
       const { distinguishedName, role } = (yield select(topicMemberRequestFormExtractor)).toJS();
-      yield call(Api.newWorkspaceMember, token, workspace.id, resource, id, role, distinguishedName);
+      yield call(Api.newWorkspaceMember, token, workspace.id, [
+        { resource, resource_id: id, role, distinguished_name: distinguishedName },
```
Comment (afoerster) [inline ui/containers/WorkspaceDetails/sagas.ts:230]: Can we do the same thing for topics?

**Review by rk-fsdev — APPROVED** [review]

File: ui/containers/WorkspaceDetails/sagas.ts (hunk 1 above)
Comment (shafi1379) [reply ui/containers/WorkspaceDetails/sagas.ts:230]: For all `Api.newWorkspaceMember` call we have already changed even for topics.

**Pull request merged**